src/ResourceManager/Profile/Commands.Profile/AutoSave/EnableAzureRmContextAutosave.cs

```diff
@@ -52,19 +52,22 @@ public override void ExecuteCmdlet()
             }
         }
 
-        void EnableAutosave(IAzureSession session, bool writeAutoSaveFile, out ContextAutosaveSettings result)
+        private static void EnableAutosave(IAzureSession session, bool writeAutoSaveFile, out ContextAutosaveSettings result)
```

Comment:
I am not sure we want to make these methods static - in general, we want methods used in cmdlet execution to be non-static methods.